### ISSUE TITLE
Dialyzer error on declare if not found

### ISSUE THREAD

**Issue by cmeiklejohn** [lasp-lang/lasp#102]
```_build/default/lib/lasp/src/lasp_plumtree_broadcast_distribution_backend.erl
 893: The pattern <{'error', 'not_found'}, {StorageId, TypeId}, {'state', Store, Actor, _, _}, Module, Function, Args> can never match the type <{'ok',{binary() | {_,_},'lasp_gcounter' | 'lasp_ivar' | 'lasp_orset' | 'lasp_pncounter' | 'riak_dt_gcounter' | {_,_},[any()],_}},binary() | {binary(),'lasp_gcounter' | 'lasp_ivar' | 'lasp_orset' | 'lasp_pncounter' | 'riak_dt_gcounter' | {'lasp_pair',[any()]} | {'lasp_top_k_set',[any()]}},#state{store::atom() | pid() | reference() | ets:tid(),actor::'undefined' | binary(),counter::'undefined' | non_neg_integer(),gc_counter::'undefined' | non_neg_integer()},'lasp_core','bind' | 'read' | 'update',[any(),...]>
 896: The created fun has no local return
```

**Comment by cmeiklejohn** [comment]
@junghunyoo Please look at this when you have a chance.

**Comment by cmeiklejohn** [comment]
Resolved by eed03db9e48831423c8e965ecc5de3696178d909.
